packages/wallet/contract/test/devices.js

```diff
@@ -2,6 +2,7 @@ import bundleCentralSupply from '@agoric/vats/bundles/bundle-centralSupply.js';
 import bundleMintHolder from '@agoric/vats/bundles/bundle-mintHolder.js';
 import bundleSingleWallet from '@agoric/vats/bundles/bundle-singleWallet.js';
 import bundleWalletFactory from '@agoric/vats/bundles/bundle-legacy-walletFactory.js';
+import bundleProvisionPool from '@agoric/vats/bundles/bundle-provisionPool.js';
```

Comment:
I'd like to stop maintaining `packages/wallet/contract`. @samsiegart @michaelfig do we need it any more?

@turadg bonus points if you beat me to it.

Reply:
https://github.com/Agoric/agoric-sdk/pull/6134/commits/22ff6e8415a88d6d42a265913cde5ae482c2fa26